Packs/AggregatedScripts/Scripts/IPEnrichment/IPEnrichment.yml

```diff
@@ -0,0 +1,394 @@
+    - 'false': use only existing TIM data and run internal commands; skip external integrations.
+    If the 'brands' argument is provided, this flag is ignored and enrichment/internal commands will run only on the brands provided.
+  auto: PREDEFINED
+  predefined:
+  - 'true'
+  - 'false'
+  defaultValue: 'false'
+  required: false
+- name: verbose
+  description: Whether to retrieve a human-readable entry for every command. When set to false, human-readable will only summarize the final result and suppress error entries from commands.
+  auto: PREDEFINED
+  predefined:
+  - 'true'
+  - 'false'
+  defaultValue: 'false'
+  required: false
+- name: brands
+  description: |
+    A list of integration brands to run enrichment against.  
+    Example: `"VirusTotal (API v3), AlienVault OTX v2"`.
+    - If provided, only the selected brands are used.
+    - If left empty, the script runs enrichment on all enabled integrations,
+      depending on the `external_enrichment` flag.
+    - In order to run get-endpoint-data add Core to the brands list.
+    - In order to run core-get-IP-analytics-prevalence, add Cortex Core - IR to the brands list.
+    To see the available brands for the `ip` command, run: `!ProvidesCommand command=ip`.
+  isArray: true
+  required: false
+- name: additional_fields
+  description: |
+    When set to true, the output will also include an `AdditionalFields` object
+    for each of the indicator result.  
+    `AdditionalFields` contains all fields returned by TIM or the integrations
+    that are not part of the standard output keys: `Address`, `DetectionEngines`, 
+    `PositiveDetections`, `Score`, and `Brand`.  
+    When set to false, only the standard keys are returned.
+  auto: PREDEFINED
+  predefined:
+  - 'true'
+  - 'false'
+  required: false
+  defaultValue: 'false'
+comment: This script gathers IP reputation data from multiple integrations and returns an IP entity with consolidated information to the context.
+commonfields:
+  id: ip-enrichment
+  version: -1
+enabled: false
+name: ip-enrichment
+outputs:
+# IPEnrichment
+- contextPath: IPEnrichment.Value
+  description: The ip address.
+  type: string
+- contextPath: IPEnrichment.MaxScore
+  description: The max score of all the indicators found.
+  type: number
+- contextPath: IPEnrichment.MaxVerdict
+  description: The max verdict of all the indicators found.
+  type: string
+- contextPath: IPEnrichment.TIMScore
+  description: The TIM score of the IP.
+  type: number
+- contextPath: IPEnrichment.Results
+  description: A list of all indicators found for the IP.
+  type: array
+- contextPath: IPEnrichment.Status
+  description: The status of the indicator. If Score changed manually, the status will be "Manual". If modified less then 1 week, the status will be "Fresh". Otherwise, the status will be "Stale". If not modified at all, the status will be None.
```

Comment:
```suggestion
  description: The status of the indicator. If the Score changed manually, the status will be "Manual". If modified by less than one week, the status will be "Fresh". Otherwise, the status will be "Stale". If not modified, the status will be None.
```